src/solvers/dgmulti/types.jl

```diff
@@ -41,6 +40,21 @@ function DGMulti(; polydeg::Integer,
                    surface_integral=SurfaceIntegralWeakForm(surface_flux),
                    volume_integral=VolumeIntegralWeakForm(),
                    kwargs...)
+
+  # call dispatchable constructor
+  DGMulti(element_type, approximation_type, volume_integral, surface_integral;
+          polydeg=polydeg, surface_flux=surface_flux, kwargs...)
+end
+
+# dispatchable constructor for DGMulti to allow for specialization
+function DGMulti(element_type::AbstractElemShape,
+                 approximation_type,
+                 volume_integral=VolumeIntegralWeakForm(),
+                 surface_integral=SurfaceIntegralWeakForm(surface_flux);
+                 polydeg::Integer,
+                 surface_flux=flux_central,
+                 kwargs...)
```

Comment:
If I understand you correctly, you mainly need this for internal use to allow dispatching. Personally, I think it's not very good practice to have default values defined in multiple, non-orthogonal places (if a person in the future decides to change one default value in one place, it might forget about the other places). Thus I suggest to remove the default values from here.
```suggestion
function DGMulti(element_type::AbstractElemShape,
                 approximation_type,
                 volume_integral,
                 surface_integral;
                 polydeg::Integer,
                 surface_flux,
                 kwargs...)
```

Reply:
That sounds like a good idea. Thanks!